CHANGES.rst

```diff
@@ -18,6 +18,8 @@ Mosviz
 Specviz
 ^^^^^^^
 
+* Re-enabled unit conversion support. [#2048]
+
```

Comment:
we might want to replace this with the eventual bulk-PR into main, but it can serve as a placeholder for now